lib/broccoli/broccoli-typescript.js

```diff
@@ -40,14 +40,13 @@ class BroccoliTypeScriptCompiler extends Plugin {
   build() {
     if (!this._tsServiceHost) {
       this._createServiceHost();
-      this._doFullBuild();
-    } else {
+      // this._doFullBuild();
```

Comment:
Is this a note for you for the future work? You could also remove the whole method since its not used :-)